devtools/server/docs/actor-hierarchy.md

```diff
@@ -63,6 +63,11 @@ and returns its `actorID`. That's the main role of RootActor.
    |   Returned by "listWorkers" request to a ChildProcessActor to get workers
    |   for the chrome of the child process.
    |
+   |-- WindowActor (window.js)
+   |   Targets a single window, such as a browser window in Firefox, but it can
+   |   be used to reach any window in the process.
+   |   Returned by "getWindow" request to the root actor.
+   |
```

Comment:
Hmm, this tree doesn't render well, not by GitHub's MarkDown renderer anyway:

https://github.com/jryans/positron/blob/browser-window-devtools/devtools/server/docs/actor-hierarchy.md

That isn't caused by your change! So can be addressed separately. But it's an easy fix, you might consider rolling it into this branch. I submitted a PR for it:

https://github.com/jryans/positron/pull/1